locator/topology.hh

```diff
@@ -236,6 +243,7 @@ public:
     // Looks up a node by its inet_address.
     // Returns a pointer to the node if found, or nullptr otherwise.
     const node* find_node(const inet_address& ep) const noexcept;
+    node* find_node(const inet_address& ep) noexcept;
```

Comment:
If absolutely needed, it should be private.

Reply:
Ok removed that method.